src/components/structural/View.js

```diff
@@ -137,9 +137,10 @@ class View extends Component {
                 return <a-text key={ent.id} {...flattened}></a-text>;
             }
             if (ent.tube) {
-                return <a-tube path={ent.path} radius={ent.radius} material={ent.material} shadow={shadow} shadowcustomsetting></a-tube>;
+                return <a-tube path={ent.path} radius={ent.radius} material={ent.material} materialtype={`type:${ent.materialtype}`} shadow={shadow} shadowcustomsetting></a-tube>;
```

Comment:
is there a specific reason why "materialtype={ent.materialtype}" doesn't work here as it does for raidus, material, etc?

Reply:
Ah, I forgot to change the JSX in the `ent.tube` path. Going to push a fix for that.

The reason I can't just pass in ent.materialtype is that this doesn't work for some reason (can't access data.type, data.roughness, etc). In order to get it to work, I had to add `type:` to the schema and split up the data into its separate components in View.js. I never really figured out why ent.material doesn't have this problem, however I also can't find out where that component is registered, so I can't really compare the two.